components/Footer.tsx

```diff
@@ -3,7 +3,7 @@ import Link from 'next/link'
 export const Footer = () => {
   return (
     <Link href="../">
-      <a>Back to main</a>
+      <a style={{paddingBottom:"50px"}}>Back to main</a>
```

Comment:
At the risk of beating a dead horse to death, avoid inline styles as a rule. Some very special times they cannot be avoided. But if you can avoid them, do. An alternate reality you who didn't avoid them and spent months of his life fighting CSS specificity wars will envy how simple your life is and how much less styling will suck for you (it will still suck, just not in a vomitous diaper blow-out kind of bad styling kind of a way).